### PR TITLE
Feature/sign in with apple

### PR DESCRIPTION
**Pull request by prasanthu**
Add Sign In with Apple to the Dev target.

1) Sign-in is the last step of Login Carousel
2) Email entry and waiting for dynamic link is not necessary
3) Dropped emailEntry & checkYourEmail stages

Pending:
-The unused storyboards and classes will be removed in another PR
-Private relay emails are not getting delivered (Issue in the backend)
-Add keys for prod target in server

### CODE REVIEW

File: ostelco-ios-client/ViewControllers/Login/LoginViewController.swift [hunk 1]
```diff
@@ -53,28 +57,46 @@ class LoginViewController: UIViewController {
...
         } else {
             primaryButton.setTitle(NSLocalizedString("Next", comment: "Action button in Carousel"), for: .normal)
+            primaryButton.isHidden = false
+            buttonStackView.isHidden = true
         }
     }
-    
+
+    func setupProviderLoginView() {
+        authorizationButton = ASAuthorizationAppleIDButton(type: .signIn, style: .whiteOutline)
```
Comment (havard024) [inline ostelco-ios-client/ViewControllers/Login/LoginViewController.swift]: I got an compiler error here, not sure why, but locally the `signUp` type does not exist for me

Screen Shot 2019-08-16 at 1 43 39 PM

File: ostelco-core/ModelControllers/UserDefaultsWrapper.swift [hunk 2]
```diff
@@ -14,6 +14,7 @@ public struct UserDefaultsWrapper {
     private enum Key: String, CaseIterable {
         case pendingEmail = "PendingEmail"
```
Comment (havard024) [inline ostelco-core/ModelControllers/UserDefaultsWrapper.swift]: I guess we could remove this, if it's only used with our firebase login

File: ostelco-ios-client/ViewControllers/Login/LoginViewController.swift (hunk 1 above)
Comment (prasanthu) [reply ostelco-ios-client/ViewControllers/Login/LoginViewController.swift]: I see a different definition 
Screenshot 2019-08-16 at 14 00 33

File: ostelco-core/ModelControllers/UserDefaultsWrapper.swift (hunk 2 above)
Comment (prasanthu) [reply ostelco-core/ModelControllers/UserDefaultsWrapper.swift]: My plan was to cleanup all unused classes, enums and storyboards in a different PR

Comment (havard024) [reply ostelco-core/ModelControllers/UserDefaultsWrapper.swift]: That makes sense 👌

File: ostelco-ios-client/ViewControllers/Login/LoginViewController.swift (hunk 1 above)
Comment (havard024) [reply ostelco-ios-client/ViewControllers/Login/LoginViewController.swift]: No issues, I didn't have the latest xcode-beta

**Pull request merged**